demos/shape.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
Rather than re-add this, can we add a pill screenshot test for button and then remove this (and remove mention of it from demos/index.html)?